example/bufferCopy/src/bufferCopy.cpp

```diff
@@ -215,10 +215,10 @@ auto main() -> int
     // padding between rows/planes of multidimensional memory allocations.
     // Therefore the pitch (distance between consecutive rows/planes) may be
     // greater than the space required for the data.
-    Idx const deviceBuffer1Pitch(alpaka::getPitchBytes<2u>(deviceBuffer1) / sizeof(Data));
-    Idx const deviceBuffer2Pitch(alpaka::getPitchBytes<2u>(deviceBuffer2) / sizeof(Data));
-    Idx const hostBuffer1Pitch(alpaka::getPitchBytes<2u>(hostBuffer) / sizeof(Data));
-    Idx const hostViewPlainPtrPitch(alpaka::getPitchBytes<2u>(hostViewPlainPtr) / sizeof(Data));
+    Idx const deviceBuffer1Pitch(alpaka::getPitchesInBytes(deviceBuffer1)[2] / sizeof(Data));
```

Comment:
`deviceBuffer1` is a 3-dimensional buffer and this line calculates the number of elements per row assuming that pitch is a multiple of the element size.
IMO this should be `Idx const deviceBuffer1Pitch(alpaka::getPitchesInBytes(deviceBuffer1)[1] / sizeof(Data));`
because the bytes to jump within the y-dimension is required to calculate number of elements per row.
The value of `getPitchBytes(buffer)[2]` for a 3-dimensional buffer is the size of an element aka how many bytes do I need to add to go to the next column.